File: erykandroidcommon/src/main/java/pl/edu/amu/wmi/erykandroidcommon/rx/StringUtils.java
package pl.edu.amu.wmi.erykandroidcommon.rx;

/**
 * @author Eryk Mariankowski <eryk.mariankowski@softra.pl> on 17.07.17.
 */
public class StringUtils {

    private StringUtils() {
    }

    public static String addLeadingZeros(int i, int len) {
        return String.format("%0" + len + "d", i);
    }


}

File: erykandroidcommon/src/main/java/pl/edu/amu/wmi/erykandroidcommon/rx/StringUtils.kt
package pl.edu.amu.wmi.erykandroidcommon.rx

/**
 * @author Eryk Mariankowski <eryk.mariankowski></eryk.mariankowski>@softra.pl> on 17.07.17.
 */
object StringUtils {

    fun addLeadingZeros(i: Int, len: Int): String {
        return String.format("%0" + len + "d", i)
    }


}
